### DISCUSSION TITLE
macOS updated Folder template Request  (Figma - Pixelmator - Photoshop)

### DISCUSSION THREAD

**Discussion by Lix35** [elrumo/macOS_Big_Sur_icons_replacements#1427] [General]
Hello All,

By Any chance does anyone have an updated  macOS Folder template for either Figma, Pixelmator, or Photoshop? 

Thank you in advance

**Comment by exp-lab-usa** [comment]
Reviving this because I never found templates. Here’s the one I made for myself in Pixelmator. It’s pretty basic but have at it. Save the vector shapes as custom shapes for easy templating or just duplicate the file each time. Surprised Apple didn’t put this in Content Hub.

[MacOS Tahoe Folder Template Export.zip](https://github.com/user-attachments/files/30620019/MacOS.Tahoe.Folder.Template.Export.zip)

<img width="1052" height="1053" alt="Screenshot - MacOS Tahoe Folder Template Export" src="https://github.com/user-attachments/assets/159109fc-9b51-433f-902d-4b75d2c8869d" />
